src/Language/Sally/Writer.hs

```diff
@@ -201,8 +203,8 @@ instance SupportTermOps (SallyReader SMT2.Term) where
 -- must substitute instances of @init.field@ with simply a naked @field@.  This
 -- is done in @structProj@.
 instance SMTLib2Tweaks a => SMTWriter (SallyWriter a) where
-  forallExpr vars t = SMT2.forall (varBinding @a <$> vars) <$> t
-  existsExpr vars t = SMT2.exists (varBinding @a <$> vars) <$> t
+  forallExpr vars t = SMT2.forall_ (varBinding @a <$> vars) <$> t
```

Comment:
I noticed that this is now depending on functionality that is only present in the `master` branch of `what4`. Should `what4` be a submodule of this repo?

Reply:
Talking with @Ptival, we've decided to just use the `what4` Hackage release instead of a submodule.

Reply:
On second thought, I forgot that `what4-transition-system` _isn't_ on Hackage. In that case, I think it would be easier just to add `what4` as a submodule dependency.

Reply:
Oh right.